jolly/jolly.py

```diff
@@ -0,0 +1,160 @@
+import discord
+from redbot.core import commands, checks
+import random
+import re
+
+BaseCog = getattr(commands, "Cog", object)
+
+holiday_prefixes = list(
+    {
+        "Abominable",
+        "Angelic",
+        "Anticipating",
+        "Aunt",
+        "Blessed",
+        "Candlelit",
+        "Cheerful",
+        "Cheery",
+        "Childlike",
+        "Chilly",
+        "Cozy",
+        "Crackling",
+        "Crisp",
+        "Crystalline",
+        "Drunk",
+        "Enchanted",
+        "Excited",
+        "Exhausted",
+        "Festive",
+        "Fireside",
+        "Frigid",
+        "Frostbitten",
+        "Frosty",
+        "Frozen",
+        "Generous",
+        "Gift-Wrapped",
+        "Gingerbread",
+        "Glacial",
+        "Glistening",
+        "Grandma",
+        "Grandpa",
+        "Happy",
+        "Heavenly",
+        "Holy",
+        "Homemade",
+        "Hypothermic",
+        "Icy",
+        "Jingling",
+        "Jolly",
+        "Joyful",
+        "Lavish",
+        "Little",
+        "Mead-Soaked",
+        "Melting",
+        "Merry",
+        "Mirthful",
+        "Naughty",
+        "Nice",
+        "Nippy",
+        "Ornamental",
+        "Pine-Scented",
+        "Roasted",
+        "Santa",
+        "Secret",
+        "Shivering",
+        "Snowbound",
+        "Snowy",
+        "Sparkling",
+        "Special",
+        "Spiced",
+        "Spiked",
+        "Sugar-Coated",
+        "Surprise",
+        "Sweet",
+        "Thankful",
+        "Thoughtful",
+        "Tickle Me",
+        "Toasty",
+        "Tranquil",
+        "Twinkling",
+        "Uncle", 
+        "Wintry",
+        "Wondrous",
+        "Wrapped",
+        "Yuletide",
+        "Yummy",
+        "Zesty"
+    }
+)
+
+holiday_suffixes = list(
+    {
+        "In a Box",
+        "the Elf",
+        "Claus",
+        "Themed Socks",
+        "the Snowman",
+        "On the Shelf",
+        "The Red-Nosed Reindeer",
+        "Incarnate"
+    }
+)
+holiday_names = list(
+    {
+        "Elsa",
+        "Mrs. Claus",
+        "Ebenezer Scrooge",
+        "Santa's Little Helper",
+        "Krampus",
+        "Rudolph",
+        "Buddy"
+    }
+)
```

Comment:
Replace these declarations with,

```python
holiday_names = [
  "Elsa",
  ...
]
```

Reply:
the `list({.......})` syntax from previous modules exists to remove duplicates in case I'd doubled up on words